app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -382,8 +382,8 @@ fun prettyTimeShortener(timeString: String): String {
                 .replace(Regex("hours?"), "h")
                 .replace(Regex("days?"), "d")
                 .replace(Regex("weeks?"), "w")
-                .replace(Regex("months?"), "M")
```

Comment:
This change is unrelated

Reply:
Sorry, I have things changed for myself and they must have gotten in here somehow. I will change it on the next commit.